project-base/storefront/components/Blocks/Banners/Banner.tsx

```diff
@@ -0,0 +1,49 @@
+import { BannerImage } from './BannerImage';
+import { CarouselState, getBannerOrderCSSProperty, getRGBColorString, getYIQContrastTextColor } from './bannersUtils';
+import { TypeSliderItemFragment } from 'graphql/requests/sliderItems/fragments/SliderItemFragment.generated';
+import { twJoin } from 'tailwind-merge';
+import { useMediaMin } from 'utils/ui/useMediaMin';
+
+type BannerProps = {
+    banner: TypeSliderItemFragment;
+    bannerSliderState: CarouselState;
+    index: number;
+    numItems: number;
+};
+
+const BannerDescription: FC<{ banner: TypeSliderItemFragment }> = ({ banner }) => (
+    <div
+        className="mt-auto p-6 lg:absolute lg:right-0 lg:h-full lg:max-w-md lg:px-14 lg:py-24"
+        style={{
```

Comment:
is this style necessary? If yes, then its ok :)

Reply:
Unfortunately yes, as I wrote in upgrade notes, tailwind cannot handle dynamic class names, so it needs to be style.

Reply:
ok then :)